async-inference/HuggingFace-Async-Inference-Walkthrough.ipynb

```diff
@@ -0,0 +1,481 @@
+{
```

Comment:
<p>Capitalization - please use <code>Hugging Face</code> ﻿for all occurrences.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2910/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Resolved